### PR TITLE
[20] Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by Jeffbuchi**
Validating the sign up form

### CODE REVIEW

**Comment by ToluAkin** [comment]
@Jeffbuchi The regex still needs to be fixed.

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done,

But, you need to write a single function that checks if an input field is empty. As you can see, the code is almost the same. The only difference is the field name and the error message you are displaying. If that is the case, then you need to use a function and pass these two things as a parameter

File: submissions/jeffersonMormah/domManipulation/signup.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+let passWord = document.getElementById("Password");
+let confPassword = document.getElementById("conf-password");
+let nameCheck = /^[A-za-z]/;
+let checkNum =  /^[\+0-9]{8,15}$/;
+let validEmail = /^+([]?+)*@+([-]?+)*({2,6})+$/;
+let pwdVal = /(?=.*\d)(?=.*[a-z])(?=.*[a-zA-Z]).{6,}$/;
+let firstNameInput = "Please Enter Your First Name";
+let firstNameError = "It can contain only letters";
+let lastNameInput = "Please Enter Your Last Name";
+let lastNameError = "Letters Only";
```
Comment (ToluAkin) [inline submissions/jeffersonMormah/domManipulation/signup.js:15]: You can reuse error message. I think you should use the `firstNameError` and rename it to be generic as `nameError` @Jeffbuchi

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Jeffbuchi Job well done. I think you should format your js file properly to enable readability. Also, your `variable names` can be `const `since the value is not going to change.

File: submissions/jeffersonMormah/domManipulation/signup.js (hunk 1 above)
Comment (Jeffbuchi) [reply submissions/jeffersonMormah/domManipulation/signup.js:15]: @ToluAkin, pls give me a hint on formating js files. Thanks

**Review by ToluAkin — APPROVED** [review]
@Jeffbuchi I hope this helps https://www.digitalocean.com/community/tutorials/understanding-syntax-and-code-structure-in-javascript

**Pull request merged**